bin/pycbc_inspiral

```diff
@@ -307,13 +308,16 @@ with ctx:
 
     tsetup = time.time() - tstart
 
+    logging.info("Performing q-transform on analysis segments")
+    out_vals['qtransform'] = qtransform.inspiral_qtransform_generator(segments)
```

Comment:
Nope doesn't actually save anything in the output. That was just my latest attempt at trying to figure out how to store the qtransform results in ref.hdf.